rst2pdf/tests/input/test_issue_274_2.rst

```diff
@@ -171,7 +171,7 @@ ffff
 * fff f fçff föfffşff fffıffıffffı füfüfff.
 * fff ff fçff ffffff fffıfı üfffffff çffışıfıfff.
 * b f İş ffffffff, c f ffffff füfffffff ffıfı ffffffffffı.
-* İş fffıfffff fffüfffı'fff fffff_ ff fe-fff_ fçff fıfff fffffüf fff.
+* İş fffıfffff fffüfffı'fff fffff ff fe-fff_ fçff fıfff fffffüf fff.
```

Comment:
I understand that the `_` at the end of the word here creates a reference and therefore was removed, but there is another one on this same line, and also another on line 262. Shouldn't they be removed as well?

Also, why was the reference pdf removed? Won't that cause the test to fail since it can't be compared to the reference?

I think you should removed these `_` and add the updated reference pdf.

Reply:
I think this was removed because it was causing a docutils error, that is normally just logged and we continue. Now that we've set rst2pdf to error if there is a docutils error, we need to remove this error from the test! @akrabat does that sound plausible?

Reply:
> I understand that the `_` at the end of the word here creates a reference and therefore was removed, but there is another one on this same line, and also another on line 262. Shouldn't they be removed as well?

Docutils doesn't detect the other two as references so they don't cause errors. I think this is because a simple reference is a trailing underscore on a set of ascii characters, so `fe-fff_` and `föfffföf_` do not qualify, but `fffff_` does.

> Also, why was the reference pdf removed? Won't that cause the test to fail since it can't be compared to the reference?

The reference PDF (rst2pdf/tests/reference/test_issue_274_2.pdf) hasn't been removed.